pkg/component/monitoring/alertmanager/component.go

```diff
@@ -36,6 +37,15 @@ const (
 	PortNameMetrics = "metrics"
 )
 
+// Interface contains functions for an alertmanager deployer.
+type Interface interface {
+	component.DeployWaiter
+	// SetIngressAuthSecret sets the ingress auth secret name.
+	SetIngressAuthSecret(*corev1.Secret)
+	// SetIngressWildcardCertSecret sets the ingress wild card cert secret name.
+	SetIngressWildcardCertSecret(*corev1.Secret)
```

Comment:
The camel case method name suggests that we see wildcard as a single word even if "wild card" also seems to be valid in the English language.

```suggestion
	// SetIngressWildcardCertSecret sets the ingress wildcard cert secret name.
	SetIngressWildcardCertSecret(*corev1.Secret)
```